proto_text.c

```diff
@@ -2067,6 +2067,31 @@ static void process_misbehave_command(conn *c) {
         out_string(c, "OK");
     }
 }
+
+static void process_debugtime_command(conn *c, token_t *tokens, const size_t ntokens) {
+    struct timeval tv;
+
+    if (strcmp(tokens[1].value, "p") == 0) {
+        gettimeofday(&tv, NULL);
```

Comment:
what's this line doing? it's checkin the clock but nothing uses tv here.

Reply:
sorry. it was just a mistake. i removed it.